Sources/GRPCProtobuf/Coding.swift

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import GRPCCore
+import SwiftProtobuf
+
+/// Serializes a Protobuf message into a sequence of bytes.
+public struct ProtobufSerializer<Message: SwiftProtobuf.Message>: GRPCCore.MessageSerializer {
+  public init() {}
+
+  /// Serializes a ``SwiftProtobuf.Message`` into a sequence of bytes.
```

Comment:
We can be more specific, `SwiftProtobuf.Message` is _any_ protobuf message, but an instance of the serializer can only serialize `Message`, a particular protobuf message.

```suggestion
  /// Serializes a ``Message`` into a sequence of bytes.
```